src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -499,6 +504,22 @@ public interface IStartConfiguration
             bool IsHeadless { get; set; }
         }
 
+        public struct CrashReporterStartupOptions
+        {
+            /// <summary>
+            /// The Autodesk CrashReport tool location on disk (directory that contains the "senddmp.exe")
+            /// </summary>
+            public string CERLocation { get; set; }
+        }
+
+        public interface IStartConfigCrashReporter
```

Comment:
TODO to get rid of this interface at 3.x?

Reply:
will add